nibabel/freesurfer/tests/test_io.py

```diff
@@ -55,21 +56,31 @@ def test_geometry():
     assert_equal(coords.shape[0], faces.max() + 1)
 
     # Test quad with sphere
```

Comment:
I hit that problem a year or two ago and managed to find one actually, feel free to use it:

https://github.com/mne-tools/mne-testing-data/tree/master/subjects/bert/surf